meshroom/core/nodes/__init__.py

```diff
@@ -0,0 +1,23 @@
+from typing import TYPE_CHECKING
+
+def getNodeClass(nodeType: str):
+    """
+    Returns the appropriate subclass of `meshroom.core.node.Node` based on `nodeType`.
+
+    Inputs
+    ------
+    nodeType: str
+        the name of the node type
+    
+    Returns
+    -------
+    type[Node]
+        the corresponding type class
+    """
+    if nodeType=="Meshing":
+        from meshroom.core.nodes.MeshingNode import MeshingNode
+        cls = MeshingNode
+    else:
+        from meshroom.core.node import Node
+        cls = Node
+    return cls
```

Comment:
The NodeDesc should be able to create a node, with a default implementation that returns "Node".
And then a NodeDesc could override the default Node without having hardcoded node names in the engine.

Reply:
Proposed solution (see commit https://github.com/alicevision/Meshroom/pull/2087/commits/ce0aa3ef7a9380eccff8a94f3a9ab225cfa1a719):
- move bounding box update handling to the Meshing node description
- give Node object reference to the corresponding node description
- add a third attribute dictionary in Node (in addition to internal and IO attributes dictionaries) that gives a flexibility to add runtime-related attributes that can be used in qml code